.github/workflows/katib_test.yaml

```diff
@@ -59,6 +59,11 @@ jobs:
     - name: Port-forward the istio-ingress gateway
       run: ./tests/gh-actions/port_forward_gateway.sh
 
+    - name: Disable Istio injection for Katib trials
```

Comment:
No, that also blocks it for all other tests, please revert immediately. Only for the Katib test

Reply:
The best way would be to disable the injection cleanly in the katib objects directly